app/helpers/remote_execution_helper.rb

```diff
@@ -194,4 +194,28 @@ def time_ago(time)
     end
   end
 
+  # Requires JS from template_invocation.js
```

Comment:
sorry I know you were told to extract it to helper but to me it sounds more natural to become a partial as this helper does not have other than rendering logic or do I miss something? maybe a helper could be only `description_format_override_checkbox_f`  and `description_format_textarea_f` that would make more sense to me